lib/summarization/entry_point.rb

```diff
@@ -67,7 +81,10 @@ def inject_into(plugin)
         #end
 
         foldable_models.each do |model|
-          plugin.register_summarization_strategy(Strategies::FoldContent.new(model))
+          DiscoursePluginRegistry.register_summarization_strategy(
```

Comment:
I think we can just return `foldable_models` in `Models::Base#available_strategies`? Nobody else consumes this register